sample/build.gradle

```diff
@@ -16,8 +16,17 @@ android {
         versionName "1.0"
     }
     buildTypes {
+        debug {
+            // remove unused classes and to make sure the proguard config is correct
+            // https://developer.android.com/studio/build/shrink-code.html#shrink-code
+            minifyEnabled true
+            // don't enable proguard, it doesn't work with instant run
+            // https://developer.android.com/studio/build/shrink-code.html#gradle-shrinker
+            useProguard false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
         release {
-            minifyEnabled false
+            useProguard true
```

Comment:
We have to use `minifyEnabled true`. Not `useProguard true` as I understand the documentation right...